components/layout/footer/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { Box, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import {
+  DiscordSVG,
+  GithubSVG,
+  HeartSVG,
+  IPXSVG,
+  TelegramSVG,
+  XSVG,
+} from '@/components/svg';
+
+const Footer: FC = () => (
+  <Box
+    fontSize="s"
+    display="flex"
+    fontFamily="Proto"
+    flexDirection="column"
+    alignItems="center"
+    textTransform="uppercase"
+    padding="4rem 0.75rem 1.5rem 0.75rem"
+  >
+    <Box
+      textAlign="center"
+      mb="l"
+      maxWidth="96%"
+      minWidth="90rem"
+      fontSize="inherit"
+      fontFamily="inherit"
+      justifyContent="center"
+    >
+      <Typography
+        size={'small'}
+        fontSize="inherit"
+        variant={'display'}
+        fontFamily="inherit"
+        lineHeight="1.25rem"
+      >
+        This DApp is maintained &
+      </Typography>
+      <Typography
+        gap="0.25rem"
+        size={'small'}
+        display="flex"
+        lineHeight={0}
+        fontSize="inherit"
+        variant={'display'}
+        fontFamily="inherit"
+        justifyContent="center"
+      >
+        built with
+        <HeartSVG maxHeight="1.125rem" maxWidth="1.125rem" width="100%" />
+        By Interest Protocol
+      </Typography>
+    </Box>
+    <Box textAlign="center" my="2xl" fontSize="inherit" fontFamily="inherit">
```

Comment:
This textAlign doesn't make sense cuz inside of component you have tag like div, instead use display.